modules/gstama/polyacleanup/main.nf

```diff
@@ -0,0 +1,37 @@
+process GSTAMA_POLYACLEANUP {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::gs-tama=1.0.3" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gs-tama:1.0.3--hdfd78af_0':
+        'quay.io/biocontainers/gs-tama:1.0.3--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fasta)
+
+    output:
+    tuple val(meta), path("*_tama.fa")                   , emit: fasta
```

Comment:
would it be possible to gzip the output fasta @sguizard

Reply:
I added the output compression on each output. But now, the conda test always fail and docker/singularity tests always succeed. I have the same result locally.

Reply:
`/tmp/tmpf9ute79l/conda/env-129b0a0d851494ee9ab3c9b8fdaf71f4/bin/tama_flnc_polya_cleanup.py: Permission denied` 

Im gettting this perm denied running with conda locally

Reply:
OK! Thanks for the review!